components/rabbitmq/scripts/creation_validation.py

```diff
@@ -17,7 +17,8 @@
     'rabbitmq_password',
     'rabbitmq_endpoint_ip',
     'rabbitmq_ssl_enabled',
-    'broker_cert_path'
```

Comment:
'broker_cert_path' is just a path and doesn't depend on inputs at all.
To check that the certs actually haven't changed, instead use 'rabbitmq_cert_public' and 'rabbitmq_cert_private'